src/store/actions/taskActions.js

```diff
@@ -0,0 +1,13 @@
+import { ADD_TASK } from "src/store";
+import { createDate } from "src/utils/helpers/utilities";
+
+export const addTask = (title) => {
+  return {
+    type: ADD_TASK,
+    payload: {
+      id: Date.now().toString(),
```

Comment:
for ID, please use uuid.

Reply:
https://github.com/azo-wtag/todo-app-react/pull/1#discussion_r1197381605